packages/react-native-payments/lib/js/PaymentRequest.js

```diff
@@ -287,41 +299,33 @@ export default class PaymentRequest {
     this._shippingOptionChangeFn(event);
   }
 
-  _getPlatformDetails(details) {
+  _getPlatformDetails(details: *) {
     return IS_IOS
       ? this._getPlatformDetailsIOS(details)
       : this._getPlatformDetailsAndroid(details);
   }
 
-  _getPlatformDetailsIOS(details: {
-    transactionIdentifier: string,
-    paymentData: string
-  }) {
+  _getPlatformDetailsIOS(details: PaymentDetailsIOSRaw): PaymentDetailsIOS {
     const {
+      paymentData: serializedPaymentData,
```

Comment:
Let's not refer to `serializedPaymentsData` as `paymentData` even we'll have a type for it --   It's likely to stump developers not using `flow`.

Reply:
Not sure I understand your advice here... Could you elaborate?